### PR TITLE
Add Rating field for Professors and TAs

### PR DESCRIPTION
**Pull request by normkoh**
Allows users to now give an optional rating from 0 to 5 to professors or TAs in their contact list.

UI for rating will be fixed in a separate PR.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -182,4 +183,19 @@ public static Location parseLocation(String location) throws ParseException {
         return new Location(trimmedLocation);
     }
 
+    /**
+     * Parses a {@code String location} into an {@code Location}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the given {@code location} is invalid.
```
Comment (JJoeYi) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:203]: JavaDocs not updated here

**Review by JJoeYi — COMMENTED** [review]
LGTM. Just a nit.

File: src/main/java/seedu/address/logic/parser/ParserUtil.java (hunk 1 above)
Comment (JJoeYi) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:203]: Also just a heads up, I just reviewed Jonas' PR, and I am worried there might be potential merge conflicts between your PR and his. Do check if there might be!

Comment (Jonaspng) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:203]: I think would have conflict. Just resove it when we merging.

**Comment by nitant-p** [comment]
I see there is no diff in the EditCommandParser file, can users still make edits for ratings?

**Review by GarethOng — COMMENTED** [review]
LGTM!

**Comment by normkoh** [comment]
@nitant-p I think I must have missed it while changing the editCommand file. Thanks for pointing that out!

**Review by nitant-p — APPROVED** [review]
LGTM!

**Pull request merged**